WordPress/Classes/ViewRelated/Basic Profile/BasicUserProfileViewController.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+final class BasicUserProfileViewController: UIViewController, DrawerPresentable, StoryboardLoadable {
+    static var defaultStoryboardName: String = "BasicUserProfile"
+
+    var expandedHeight: DrawerHeight = DrawerHeight.intrinsicHeight
+
+    let allowsUserTransition = false
+
+    @IBOutlet private weak var gravatarImageView: CircularImageView!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var usernameLabel: UILabel!
+    @IBOutlet private weak var locationLabel: UILabel!
+    @IBOutlet private weak var aboutMeLabel: UILabel!
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var sitesTableView: UITableView!
+
+    var viewModel: BasicUserProfileViewModel!
+
+    @objc class func present(context: UIViewController, view: UIView, email: String?, avatarURL: URL?) {
```

Comment:
would it be possible to move this in an extension? This way, if for any reason we need to move away the presentation from the class itself (e.g. in a coordinator) it would be easier.

Reply:
Sure